lib/main.dart

```diff
@@ -1,67 +1,315 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-
+import 'package:flutter/services.dart';
 void main() {
-  runApp(const MyApp());
+  WidgetsFlutterBinding.ensureInitialized();
+  SystemChrome.setPreferredOrientations([DeviceOrientation.landscapeLeft])
+      .then((_) {
+  runApp(MyApp());
+  });
 }
 
+var marginDivider = 1;
```

Comment:
Avoid using global variables. While it looks simple and harmful, it is hard to trace and debug when app grows. Usually, your class should only operate with it's own private fields + what is passed as paratemeters to it's method. If you need multiple classees to communicate - they should do it using parameters passing (not global variables)